data/data/azure/vnet/public-lb.tf

```diff
@@ -46,10 +48,28 @@ resource "azurerm_lb_rule" "public_lb_rule_api_internal" {
   enable_floating_ip             = false
   idle_timeout_in_minutes        = 30
   load_distribution              = "Default"
-  probe_id                       = azurerm_lb_probe.public_lb_probe_api_internal.id
+  probe_id                       = azurerm_lb_probe.public_lb_probe_api_internal[0].id
+}
+
+resource "azurerm_lb_rule" "internal_outbound_rule" {
```

Comment:
I think we can no create this at all.. we just need the virual machines to be in the backend of the public load balancer.